### ISSUE TITLE
Quotas form in Admin panel doesn't populate options

### ISSUE THREAD

**Issue by davelopez** [galaxyproject/galaxy#13597]
**Describe the bug**

**Galaxy Version and/or server at which you observed the bug**
Galaxy Version: 22.01
Commit: 3231165d7c3d9b93dfa5d3c5b99f8550d21fc3bf

**To Reproduce**
1. Make sure you have quotas enabled in your config
2. Log in as Admin
3. Go to http://127.0.0.1:8080/admin/form/create_quota
4. Observe that the available options are not populated for the `operation` and `default` quota fields in the form

**Expected behavior**
The options should appear in a dropdown selector as previous versions of Galaxy

**Screenshots**

![Screenshot from 2022-03-23 13-33-52](https://user-images.githubusercontent.com/46503462/159700426-118dc0f8-a474-4f2a-bcc8-9b940a5dfc2a.png)

**Comment by natefoo** [comment]
Just hit this as well, and if you click "save" with either field blank, you get a generic 500 error in the UI and this in the logs:

```pytb
galaxy.web.framework.decorators ERROR 2022-04-04 14:01:02,213 [pN:main.web.5,p:29079,w:5,m:0,tN:uWSGIWorker5Core2] Uncaught exception in exposed API method:
Traceback (most recent call last):
  File "/cvmfs/main.galaxyproject.org/galaxy/lib/galaxy/web/framework/decorators.py", line 190, in decorator
    rval = func(self, trans, *args, **kwargs)
  File "/cvmfs/main.galaxyproject.org/galaxy/lib/galaxy/web/framework/decorators.py", line 112, in decorator
    return func(self, trans, *args, **kwargs)
  File "/cvmfs/main.galaxyproject.org/galaxy/lib/galaxy/webapps/galaxy/controllers/admin.py", line 730, in create_quota
    quota, message = self.quota_manager.create_quota(payload, decode_id=trans.security.decode_id)
  File "/cvmfs/main.galaxyproject.org/galaxy/lib/galaxy/managers/quotas.py", line 44, in create_quota
    params = CreateQuotaParams.parse_obj(payload)
  File "pydantic/main.py", line 511, in pydantic.main.BaseModel.parse_obj
  File "pydantic/main.py", line 331, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for CreateQuotaParams
default
  value is not a valid enumeration member; permitted: 'unregistered', 'registered', 'no' (type=type_error.enum; enum_values=[<DefaultQuotaValues.UNREGISTERED: 'unregistered'>, <DefaultQuotaValues.REGISTERED: 'registered'>, <DefaultQuotaValues.NO: 'no'>])
```

Because they're not populated, you have to know what the valid values are.

**Comment by davelopez** [comment]
Fixed on #13639
